src/main/webapp/Dialogs/form.js

```diff
@@ -0,0 +1,14 @@
+function showForm() {
+    const formTemplate = document.querySelector("#demo-form");
+    const form = formTemplate.firstElementChild.cloneNode(true);
+    const toggle = document.getElementById("formsubmit");
+    const title = formTemplate.dataset.title;
+    dialog.form(form, {title: title, okText: "Order", submitButton: toggle.checked}).then(formData => {
```

Comment:
I would suggest some line wrapping to reduce scroll bars in code examples (may want to verify how this looks locally for formatting):
```suggestion
    dialog.form(form, {
      title: title, okText: "Order", submitButton: toggle.checked
     }).then(formData => {
```